.github/workflows/build-test.yaml

```
@@ -28,6 +28,16 @@ jobs:
           args: --timeout 10m --exclude SA5011 --verbose --issues-exit-code=0
           only-new-issues: true
 
+  govulncheck:
+    runs-on: ubuntu-latest
+    name: Run govulncheck
+    steps:
+      - id: govulncheck
+        uses: golang/govulncheck-action@v1
+        with:
+           go-version-input: 1.22
```

Wouldn't this be better? so that go.mod is updated and the version's updated, instead of updating in multiple places?

```suggestion
           go-version-file: 'go.mod'
```